### PR TITLE
Add configuring section to README

### PR DESCRIPTION
**Pull request by gandarez**
This PR adds a detailed `configuring` section to README.

### CODE REVIEW

**Review by dron22 — COMMENTED** [review]
@alanhamlett talked about README consisting of links, instead of text (https://wakatime.slack.com/archives/GUWD67UE4/p1619990137009100?thread_ts=1619984948.002900&cid=GUWD67UE4). So what is the plan regarding this?

**Comment by gandarez** [comment]
`CONFIGURING.md`?

**Comment by dron22** [comment]
> `CONFIGURING.md`?

Sounds good :+1:

**Comment by gandarez** [comment]
@alanhamlett would you like to contribute to this PR?

File: CONFIGURING.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+| include_only_with_project_file | Disables tracking folders unless they contain a `.wakatime-project file`. | true;false |
+| exclude_unknown_project        | When set, any activity where the project cannot be detected will be ignored. | true;false |
+| status_bar_enabled             | Turns on wakatime status bar for certain editors. | true;false |
+| status_bar_coding_activity     | Prints today's coding activity. | true;false |
+| offline                        | Enables offline mode. All activity and logged time will be queued. | true;false |
+| proxy                          | Optional proxy configuration. Supports HTTPS and SOCKS proxies. | `https://user:pass@host:port` or `socks5://user:pass@host:port` or `domain\\user:pass` |
+| no_ssl_verify                  | Disables SSL certificate verification for HTTPS requests. By default, SSL certificates are verified. | true;false |
+| ssl_certs_file                 | Override the bundled Python Requests CA certs file. By default, uses  system ca certs. | _filepath_ |
```
Comment (dron22) [inline CONFIGURING.md:54]:
```suggestion
| ssl_certs_file                 | Override the bundled CA certs file. By default, uses system ca certs. | _filepath_ |
```

File: CONFIGURING.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+| exclude                        | Filename patterns to exclude from logging. POSIX regex syntax. | true;false;regex list |
+| include                        | Filename patterns to log. When used in combination with `exclude`, files matching `include` will still be logged. POSIX regex syntax | true;false;regex list |
+| include_only_with_project_file | Disables tracking folders unless they contain a `.wakatime-project file`. | true;false |
+| exclude_unknown_project        | When set, any activity where the project cannot be detected will be ignored. | true;false |
+| status_bar_enabled             | Turns on wakatime status bar for certain editors. | true;false |
+| status_bar_coding_activity     | Prints today's coding activity. | true;false |
+| offline                        | Enables offline mode. All activity and logged time will be queued. | true;false |
+| proxy                          | Optional proxy configuration. Supports HTTPS and SOCKS proxies. | `https://user:pass@host:port` or `socks5://user:pass@host:port` or `domain\\user:pass` |
```
Comment (dron22) [inline CONFIGURING.md:52]: `domain\\user:pass` is actually the NTLM syntax. Although we should probably verify that NTLM is working in IRL. I stick to the specification, when implementing it, but do not have access to a NTLM proxy server.

@alanhamlett we earlier talked about approaching some wakatime users, to verify that NTLM working. Do you want to reach out to some?
```suggestion
| proxy                          | Optional proxy configuration. Supports HTTPS, SOCKS and NTLM proxies. | `https://user:pass@host:port` or `socks5://user:pass@host:port` or `domain\\user:pass` |
```

File: CONFIGURING.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+# Configuring
+
```
[the remark this answers is not in the document]
Comment (gandarez) [reply CONFIGURING.md:2]: yes sure.

File: CONFIGURING.md (hunk 1 above)
Comment (gandarez) [reply CONFIGURING.md:54]: nice catch also updated the flag description.

**Review by dron22 — APPROVED** [review]
LGTM :+1:

**Comment by gandarez** [comment]
Will be tackled by https://github.com/wakatime/wakatime-cli/pull/376

**Pull request closed without merging**